src/hal/i_components/latencybins.icomp

```diff
@@ -25,6 +25,10 @@ Usage:
    The latency pin outputs the instantaneous latency.
 
 Maintainers note: hardcoded for MAXBINNUMBER==1000
+
+NB.  Since there is no hal_64_t yet, this component has been left 'as is'
+using __64 type etc
```

Comment:
rtapi_get_clocks returns long long int, _s64 could be used internally by the component, the rest could still be converted to hal data types